notifications_utils/recipients.py

```diff
@@ -362,7 +365,13 @@ def __init__(
         placeholders,
         template,
         allow_international_letters,
+        validate_row=True,
     ):
+        # if we aren't validating the row, then no need to store template (used to check message length) and the cell
+        # level stuff (checks phone nums are valid, checks placeholders are present, etc)
```

Comment:
Very optional, could be a little bit clearer, in particular around 'cell level stuff' which might be better as `Cell` to show it is the class you are referring to and removing 'stuff' which is vague

If we don't need to validate, then:
- by setting no template we avoid the template level validation (used to check message length)
- by setting no `error_fn`, we avoid the `Cell` level validation (used to check phone nums are valid, placeholders are present, etc)

Does make me wonder if `validation_fn` would be a better name instead of `error_fn` for `Cell.__init__` but that is probably one for another day.